app/controllers/queue_items_controller.rb

```diff
@@ -17,11 +17,46 @@ def create
 
     redirect_to user_queue_items_path(current_user)
   end
+require 'pry'
+  def update
```

Comment:
`update` name usually points that 1 object is being updated. You should follow solution in this and stick to something like `update_items` in that sense. Its not following REST style!